Jenkinsfile

```diff
@@ -1,3 +1,26 @@
+def get_build_info() {
+	pr_branch = ''
+	if (env.CHANGE_BRANCH != null) {
```

Comment:
What if it is `null`?

Reply:
I played around with this doing replays.

It's the difference between this (not null):
Build #1 of lisk-template/PR-105 (95-jenkins-slack) failed (console, changes)

and this (null):
Build #2 of lisk-template/PR-105 failed (console, changes)

Reply:
`env.CHANGE_BRANCH` will be `null` when building a branch (as opposed to a PR)